### ISSUE TITLE
Ranger doesn't save command

### ISSUE THREAD

**Issue by hektr** [ranger/ranger#1219]
If you run some shell command from ranger, using shift+@ ranger(or shell?) doesn't save this command. So, if you need to run the same command second time, you should type it second time.

But if you run shell command using s, it works fine. You can even run shell using s and type %s by hands and this also works fine)) Only shift + @ does have problem :)

**Comment by toonn** [comment]
You're right, completion only works if whatever text is on the console is a prefix. I don't see a problem relaxing this to a fuzzy match except for maybe performance.

**Comment by hektr** [comment]
hi! It will work if you delete %s from string. So, problem is caused by existing %s on string ^_^ Ranger save command, but can't put it in string, when you press arrows, if %s appears on start string. Hope, this information will help you ^_^
